src/fonts/text.rs

```diff
@@ -99,21 +90,46 @@ where
     S: TextStyle<Color = C>,
 {
     fn bounding_box(&self) -> Rectangle {
-        self.style.bounding_box(&self.primitive)
+        let mut position = self.primitive.position;
+
+        let mut min_max: Option<(Point, Point)> = None;
+
+        for line in self.primitive.text.lines() {
+            let (bounding_box, position_delta) = self.style.bounding_box(line, position);
+
+            if let Some(bottom_right) = bounding_box.bottom_right() {
+                if let Some((min, max)) = &mut min_max {
+                    min.x = min.x.min(bounding_box.top_left.x);
```

Comment:
Preparing for RTL and bottom-to-top text? :) I would assume `min == position` otherwise, is there something I don't see?

Reply:
I wanted to keep it as generic as possible. And I think this will be required if we add horizontal alignment support, even without RTL.